test/sparse/sparse.jl

```diff
@@ -1263,66 +1344,69 @@ let smalldim = 5, largedim = 10, nzprob = 0.4, targetnumposzeros = 5, targetnumn
             @test length(dropzeros!(copy(Awithzeros), false).nzval) == length(Awithzeros.nzval)
             @test length(dropzeros!(copy(Awithzeros), false).rowval) == length(Awithzeros.rowval)
         end
+        # original lone dropzeros test
```

Comment:
this doesn't need to be inside the `for (m, n)` loop since it doesn't depend on either `m` or `n`